no-computer.md

```diff
@@ -1,44 +1,89 @@
 # No computer project
 
+_Note: Since these are project proposals but this file is specifically not_
+_supposed to be computer related, I assume this is more about arguing that_
+_something is a DSL and not about actually building it. (i.e. I don't know_
+_if I can actually build a bike.) Sorry if thisanswer is a little too jokey!_
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This language, a bike, would serve people who need to get around easier.
+Here, we are assisting any able body person with a need to get somewhere
+via self-propelled land vehicle. Although one may note land vehicles such as
+cars, horses, and regular bikes already exist, one may note that these usually
+primarily only provide exercise to a user's leg, if even. What if someone
+wants to get somewhere, be fit, and skip leg day all at the same time? This
+idea, a bike that pedals with your arms (a.k.a. the __ARM BIKE__), can allow
+for all the convenience of a typical land vehicle, while enabling a strong
+upper body workout.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+When someone wants to go somewhere, they have a desire that they want to
+express. Primarily, this language manifests itself verbally as simple commands
+like "Go forward!", "Go backward!", "Stop!", and "Turn!" That being said,
+the verbal DSL does not allow these desires to be fufilled. By using a great
+interpreter like the __ARM BIKE__, these desires can be expressed as
+physical commands that can then be executed in _real life_.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+It is a __BIKE__ that you use __WITH YOUR ARMS!__ What's not exciting about
+that? I came up with this idea because I was thinking how disproportionally
+strong my legs are relative to my upperbody. I generally find most upperbody
+workouts are boring and would benefit greatly from something like this that
+I could incoperate into my daily life by switching to the __ARM BIKE__ instead
+of a regular bike.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Slow biking using feeble arms.
```

Comment:
Really funny! Not too much to comment here, because it is (as you said) fairly jokey in nature, but I do think this is a neat idea.